activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/test/java/org/activiti/cloud/services/events/listeners/CloudApplicationDeployedProducerTest.java

```diff
@@ -0,0 +1,107 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.events.listeners;
+
+import org.activiti.api.process.model.Deployment;
+import org.activiti.api.process.model.events.ApplicationDeployedEvent;
+import org.activiti.api.runtime.event.impl.ApplicationDeployedEventImpl;
+import org.activiti.api.runtime.event.impl.ApplicationDeployedEvents;
+import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
+import org.activiti.cloud.api.model.shared.impl.events.CloudRuntimeEventImpl;
+import org.activiti.cloud.api.process.model.events.CloudApplicationDeployedEvent;
+import org.activiti.cloud.services.events.ProcessEngineChannels;
+import org.activiti.cloud.services.events.converter.RuntimeBundleInfoAppender;
+import org.activiti.cloud.services.events.message.MessageBuilderAppenderChain;
+import org.activiti.cloud.services.events.message.RuntimeBundleMessageBuilderFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.support.MessageBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class CloudApplicationDeployedProducerTest {
+
+    @InjectMocks
+    private CloudApplicationDeployedProducer cloudApplicationDeployedProducer;
+
+    @Mock
+    private RuntimeBundleInfoAppender runtimeBundleInfoAppender;
+
+    @Mock
+    private ProcessEngineChannels producer;
+
+    @Mock
+    private MessageChannel auditProducer;
+
+    @Mock
+    private RuntimeBundleMessageBuilderFactory runtimeBundleMessageBuilderFactory;
+
+    @Mock
+    private MessageBuilderAppenderChain messageBuilderAppenderChain;
+
+    @Captor
+    private ArgumentCaptor<CloudRuntimeEvent<?, ?>[]> messagePayloadCaptor;
+
+    @BeforeEach
+    public void setUp() {
+        initMocks(this);
+        when(producer.auditProducer()).thenReturn(auditProducer);
+        when(runtimeBundleMessageBuilderFactory.create()).thenReturn(messageBuilderAppenderChain);
+    }
+
+    @Test
+    public void shouldSendMessageWithDeployedApplicationWhenWebApplicationTypeIsServlet() {
```

Comment:
The suffix `WhenWebApplicationTypeIsServlet` seems wrong here as we are not checking the web application type in the code. I think can rename it to `should_sendMessageWithDeployedApplication`.